scripts/VISS_Sample_Data.R

```diff
@@ -0,0 +1,357 @@
+library(tidyverse)
```

Comment:
Shouldn't this go into the `R` folder?

Reply:
I see, makes sense!

Reply:
Added https://github.com/uw-ssec/biodiversity-horizons/issues/22 to refactor this after the PR is merged